glimmer-version/app/components/focusing-inner.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  didUpdateAttrs() {
+    this._super(...arguments);
+    this.attrs.focus(this.get('outletState'))
```

Comment:
For some reason I feel like this was broken without using `.attrs` in some version. Agreed in principle, reality I believe dictated this approach.